08-rails/routes-controllers.md

```diff
@@ -45,6 +45,77 @@ Let's break down this code:
   - **request method**: `get` in this example
   - **path**: what will be matched with the URL in the HTTP request, `"/books"` in this example
   - **controller and action**: defining the controller and action, split by the `#`. `"books#index"` in this case this would point to the `index` method in the `BooksController` class.
+  
+### Naming Routes 
+
+A `routes.rb` file which lists all the CRUD operations would look like this:
+
+```ruby
+# config/routes.rb
+Rails.application.routes.draw do
+
+  get "/books/new", to: "books#new", as: "new_book"
+  post "/books", to: "books#create"
+
+  get '/books/:id/edit', to: 'books#edit', as: "edit_book"
+  patch '/books/:id', to: 'books#update'
+
+  get "/books", to: "books#index"
+  get "/books/:id", to: "books#show", as: "book"
+
+  delete "/books/:id", to: "books#destroy", as: "delete_books"
+end
+```
+
+You may be saying, "Wait what's that `as: <STUFF>` part?"  This allows us to name a path.  It's very helpful to give each path for a resource, like books, a name.  The name can then be used later in your code to link actions together without having to give a hardcoded path like `'/books/new'` each time.  That way if we change a path, we only have to edit the `routes.rb` file instead of each line in our code where we use the route.  You can think of it like a constant variable referencing each path.  So `edit_book` is the name of the `/books/:id/edit` path.  
```

Comment:
I'm not sure its helpful to say that we can think of the route helpers like constants as they are methods and students will be required to use them like methods whenever parameters are required.